src/authorization/AuthenticationService.ts

```diff
@@ -47,8 +47,11 @@ export class AuthenticationService implements IAuthenticationService {
    */
   public isAuthenticated(
     request: ccfapp.Request<any>,
-  ): [ccfapp.AuthnIdentityCommon, ServiceResult<string>] {
-    let caller: ccfapp.AuthnIdentityCommon;
+  ): [
+    ccfapp.AuthnIdentityCommon | undefined,
+    ServiceResult<string> | undefined,
```

Comment:
Shouldn't we always return `ServiceResult<string>`? I think we want `validator.validate(request)` without `?` after confirming `validator` is not null/undefined.

Reply:
I added a failed result so undefined is never returned.